libs/renterd-types/src/types.ts

```diff
@@ -204,6 +198,14 @@ export type PricePinSettings = {
   gougingSettingsPins: GougingSettingsPins
 }
 
+// Contracts
+
+export type ContractSpending = {
```

Comment:
We might be missing some spending fields here, this 'll be the new spending fields after we merge soft deletion into the `api-breakers` branch:

```
ContractSpending struct {
		Deletions   types.Currency `json:"deletions"`
		FundAccount types.Currency `json:"fundAccount"`
		SectorRoots types.Currency `json:"sectorRoots"`
		Uploads     types.Currency `json:"uploads"`
	}
```

Reply:
No more downloads?

Reply:
Download spending is gone because we don't use contracts to pay for downloads anymore. So that spending is included in `FundAccount`.
We'll need to add a new `AccountSpending` at some point which breaks down the account spending like we do the contract spending now.